### DISCUSSION TITLE
[Compatibility] Graphit Processing ModCollision

### DISCUSSION THREAD

**Discussion by Flagran** [Xman1109/Atomic_Overhaul#15] [General]
Hey  X_Man1109,

i try to play your intresting mod with K2/SE und Brevvens mods its works fine but i discovered a issus with the Graphit Processing. In Brevvens mod "Graphite & Diamonds" (https://mods.factorio.com/mod/bzcarbon) its excatly the same research "Graphit Processing". Yours overwride the other one. Thats why i have to revoke your mod from my modlist. :(

Can you please take a look about this issus?? 

By the way. Its possible to make all Graphitcells from your mod Optional via ModConfig? Its sound a bit strange that i can make power from Graphit. 

Greetings

Flaggi

**Comment by Xman1109** [comment]
In version 1.2.6 I tried adding compatibility for the exact same mod, which at that point worked fine, but there probably was an update which broke something…
I will investigate and fix this issue, which probably takes less than 24h.
Thank you for reporting this.

Greetings
X_Man1109

**Comment by Xman1109** [comment]
Could you tell me what exactly is getting overwritten?
![image](https://user-images.githubusercontent.com/72877576/188656530-1095f226-9f46-4d8e-8eca-7fbbffc498d7.png)
I can't see where AO overwrites some other technology.

**Comment by Flagran** [comment]
Okay, after a bit Testing i have a the solution.

after Adding your mod zu my savegame is discoverd that the old Graphit Processing Research 

![grafik](https://user-images.githubusercontent.com/90524369/188666359-06001f41-b51b-4088-95b2-31b7dbd8c7b2.png)

are replaced with yours 

![grafik](https://user-images.githubusercontent.com/90524369/188665744-1d019f73-5f2d-46f9-9136-a0f895aaa922.png)

i had to Research it again so i can grind GraphitFlakes to Graphit once again.

![grafik](https://user-images.githubusercontent.com/90524369/188667172-7a02f79f-505e-44d9-af64-25972dedba74.png)

now its functional.

Greetings 

Flaggi

**Reply by Xman1109** [reply]
Oh yeah, I had to rename my tech, so it would not overwrite the other one from Brevvens mod. You should only need to research this tech once after adding the mod to your game, if otherwise, please tell me, so I will try to fix it. But for now I will be closing this issue.

**Comment by Setos79** [comment]
Good afternoon. I discovered a small visual bug associated with this mod.
In crafting Graphite, icons that are not entirely correct are used, which are a little misleading.
![image](https://github.com/Xman1109/Atomic_Overhaul/assets/144532245/bc38576a-ed0a-4c4d-b589-45f34c7d9f15)
